docs/tutorial.rst

```diff
@@ -3,6 +3,96 @@
 ========
 Tutorial
 ========
+There are two main ways of accessing the resources of the ``stdpopsim`` package that will
+be detailed in this tutorial. The first is via the command line interface (CLI). This is
+useful if you want to do a straightforward run of the models in the ``stdpopsim``
+catalog. The other way to access the ``stdpopsim`` resources is via the python API. This
+is a bit more complicated however it allows for more advanced tasks. This tutorial will
+walk through both ways of using the ``stdpopsim`` package as well as some more advanced
+tasks you may wish to do.
+
+***************************
+Using the ``stdpopsim`` CLI
+***************************
+The CLI provides access to the catalog of models that have already been implemented by
+``stdpopsim``. The first step for using the CLI is to select the species that you are
+interested in simulating data for. In order to see which species are available run
+
+.. code-block:: console
+
+    $ stdpopsim --help
+
```

Comment:
I think it's helpful to also show what the output will look like, so someone following along to match it to what they have.
Although, since this is a static tutorial, it will get out of date.

Something like:
```
This will give something like:

.. code-block:: console

    usage: stdpopsim [-h] [-V] [-v] [-c CACHE_DIR] [-e {msprime}]
                     {homsap,aratha,esccol,dromel,download-genetic-maps} ...

    Command line interface for stdpopsim.

    positional arguments:
      {homsap,aratha,esccol,dromel,download-genetic-maps}
        homsap              Run simulations for Homo sapiens.
        aratha              Run simulations for Arabidopsis thaliana.
        esccol              Run simulations for Escherichia coli.
        dromel              Run simulations for Drosophila melanogaster.
        download-genetic-maps
                            Download genetic maps

This shows the species currently supported by ``stdpopsim``, (e.g. ``homsap``,
``arath``,...). 
```

Reply:
I talked with @jeromekelleher before and we decided against this but I'm open to it if he's convinced.

Reply:
It does look good, but this will get out of date so quickly, especially in these early stages. Let's leave it out for now, and maybe when things have settled down we can stick them in.